recipe/conda_build_config.yaml

```diff
@@ -1,6 +1,3 @@
-c_stdlib_version:
-  - 2.17
```

Comment:
We still need this because this recipe contains binary redists. Thus, we need to declare the minimum version that the redist binaries support.

Reply:
Thanks, I will add it back. In fact one of the libraries has requirement on glibc2.27. There used to be comments in the prior recipe that conda-forge infrastructure doesn't support glibc>2.17. Is that still relevant?

Reply:
The presently infrastructure supports 2.17, 2.28, and 2.34.

Reply:
Would it be appropriate to correctly reflect the dependency on glibc>=2.27 by choosing 2.28? Until the cutensor dependencie's pinning is fixed, we can still explicitly require glibc=2.17 at test time and skip the tests which break due to outdated glibc?